db/schema.rb

```diff
@@ -51,15 +51,15 @@
   end
 
   create_table "publication_reports", force: :cascade do |t|
-    t.integer  "publication_id",                     null: false
-    t.integer  "publication_version",                null: false
+    t.integer  "publication_id",                    null: false
+    t.integer  "publication_version",               null: false
     t.integer  "report"
-    t.string   "active_device_dev_uuid", limit: 64,  null: false
+    t.string   "active_device_dev_uuid", limit: 64, null: false
```

Comment:
**note for self**
we need to change this, doesn't have anything to do with this branch but it's 256 right now